plugins/base/src/main/resources/dokka/scripts/platformContentHandler.js

```diff
@@ -7,24 +7,47 @@ window.addEventListener('load', () => {
     document.querySelectorAll("div[data-platform-hinted]")
         .forEach(elem => elem.addEventListener('click', (event) => togglePlatformDependent(event,elem)))
     document.querySelectorAll("div[tabs-section]")
-        .forEach(elem => elem.addEventListener('click', (event) => toggleSections(event)))
+        .forEach(elem => elem.addEventListener('click', (event) => toggleSectionsEventHandler(event)))
     document.getElementById('filter-section').addEventListener('click', (event) => filterButtonHandler(event))
     initializeFiltering()
     initTabs()
+    handleAnchor()
 })
 
+function handleAnchor() {
+    let searchForTab = function(element) {
+        if(element && element.hasAttribute) {
+            if(element.hasAttribute("data-togglable")) return element;
+            else return searchForTab(element.parentNode)
+        } else return null
+    }
+    let anchor = window.location.hash
+    if (anchor != "") {
```

Comment:
you can just do `if(anchor)` or i'd recommend to go for triple equals so: `!==` since it checks the type